app/controllers/middleware_domain_controller.rb

```diff
@@ -9,6 +9,13 @@ class MiddlewareDomainController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
+  OPERATIONS = {
+    :middleware_domain_stop => {:op   => :stop_middleware_server,
```

Comment:
I'm reusing the operation. Is the same for middleware_server and middleware_domain